ProjectTemplates/templates/StarterWeb-CSharp/appsettings.json

```diff
@@ -0,0 +1,42 @@
+{
+////#if (IndividualB2CAuth)
```

Comment:
can this be deleted?

Reply:
No we should not remove them, and they won't appear in the generated project. 
Those are special comments which are about conditionnally generating code based on the parameters. 

What happens is that when the user writes:
`dotnet new mvc -auth IndividualB2C`, this defines the `IndividualB2CAuth` variable, and then the dotnet new template generation engine will generate the files (here the appsettings.json) conditionnally based on these variables. This section will appear in the appsettings.json if the user wants a B2C application

This is pretty awesome technology, BTW. See https://github.com/dotnet/templating/wiki for details